pkg/tools/cli/loadtest/assets/loader.go

```diff
@@ -149,10 +166,11 @@ func LoadDataWithRandomIDs(path string) (ids []string, train, test, distances []
 	return CreateRandomIDs(len(train)), train, test, distances, neighbors, dim, nil
 }
 
-func LoadDataWithSequentialIDs(path string) (ids []string, train, test, distances [][]float32, neighbors [][]int, dim int, err error) {
+// LoadDataWithSerialIDs returns approximate nearest neighbor benchmark dataset with serial IDs.
+func LoadDataWithSerialIDs(path string) (ids []string, train, test, distances [][]float32, neighbors [][]int, dim int, err error) {
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
How about to return Dataset or dataset struct?
```suggestion
func LoadDataWithSerialIDs(path string) (ds Dataset, err error) {
```
```suggestion
func LoadDataWithSerialIDs(path string) (ds dataset, err error) {
```